### ISSUE TITLE
Add public profile page of user

### ISSUE THREAD

**Issue by xndrdev** [JustServerless/discuss#26]
Just an overview of the profile of a user with all his posts and stuff like that.

**Comment by pmuens** [comment]
Good idea. We can nicely leverage the nesting feature of GraphQL for that!
